pom.xml

```diff
@@ -85,10 +86,11 @@
     <module>orientdb</module>
     <module>redis</module>
     <module>voldemort</module>
-    <module>distribution</module>
     <!--<module>mapkeeper</module>-->
     <!--module>nosqldb</module-->
     <module>couchbase</module>
+    <module>tarantool</module>
+    <module>distribution</module>
```

Comment:
I've tried to build without changing `<module>distribution</module>` position, but i can't, tarantool/couchbase jar weren't in .tar.gz file.